composer.json

```diff
@@ -47,6 +48,7 @@
     "require-dev": {
         "ext-xdebug": "*",
         "adlawson/vfs": "dev-develop",
-        "phpunit/phpunit": "^9.0"
+        "phpunit/phpunit": "^9.0",
+        "php-etl/pipeline": "^0.4.0"
```

Comment:
With a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php), there is no need for php-etl/pipeline